### PR TITLE
Build for EAP and Canary

### PR DESCRIPTION
**Pull request by stevemessick**
Update build to produce a single zip that works in IJ EAP and AS Canary IDEs.

After this is merged source builds need to use Dart plugin sources on branch 183.2153 of intellij-plugins. The Android Studio sources are sync'd to build 5081642.

@terry @jacob314 @devoncarew

### CODE REVIEW

File: flutter-intellij.iml [hunk 1]
```diff
@@ -58,5 +58,6 @@
     </orderEntry>
     <orderEntry type="library" name="Dart SDK" level="project" />
     <orderEntry type="library" name="Dart Packages" level="project" />
+    <orderEntry type="library" name="netty-codec-http" level="project" />
```
Comment (devoncarew) [inline flutter-intellij.iml]: Is this change needed? I don't see any references in Java files.

**Comment by devoncarew** [comment]
> After this is merged source builds need to use Dart plugin sources on branch 183.2153 of intellij-plugins.

Let me know if we need any new uploads to google storage -

**Comment by stevemessick** [comment]
If we merge this PR then anyone working with AS sources would need to
switch to the latest Canary version. I'm not so sure that I want to go that
route. I'm thinking about rewriting the edits to be applied to the Beta
version, when compiling for EAP/Canary. That would keep us on a more stable
source base for the next couple months.

The change to the netty library is required when compiling from an EAP
base. The Dart plugin already does it that way.

And thanks, but I already uploaded the new AS binary.

**Comment by stevemessick** [comment]
I'm going to optimistically merge this. If it causes too many problems then it will be reverted.
